maskrcnn_benchmark/config/defaults.py

```diff
@@ -221,6 +221,75 @@
 _C.MODEL.RESNETS.RES2_OUT_CHANNELS = 256
 _C.MODEL.RESNETS.STEM_OUT_CHANNELS = 64
 
+
+# ---------------------------------------------------------------------------- #
+# RetinaNet Options (Follow the Detectron version)
+# ---------------------------------------------------------------------------- #
+_C.RETINANET = CN()
```

Comment:
I think from discussions with @rbgirshick we might want to add everything model-related to the `_C.MODEL` namespace.
Could you rename this to
```python
_C.MODEL.RETINANET = CN()
```
?